### PR TITLE
ergodox-zweihander-osx: initial commit

### PR DESCRIPTION
**Pull request by adiabatic**
This is my layout. This name came to me when I was thinking about my mouse-and-keyboard navigation style. While it may only be directly useful to a tiny handful of lefthanded OS X users, I think I have some neat ideas about what goes in a media layer.

### CODE REVIEW

File: keyboard/ergodox_ez/keymaps/zweihander-osx/keymap.c [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+/* Keymap 0: Basic layer
+ *
+ * ,--------------------------------------------------.           ,--------------------------------------------------.
+ * |   =    |   1  |   2  |   3  |   4  |   5  |   ←  |           |   →  |   6  |   7  |   8  |   9  |   0  |   -    |
+ * |--------+------+------+------+------+-------------|           |------+------+------+------+------+------+--------|
+ * |   ⌦    |   Q  |   W  |   E  |   R  |   T  |  L1  |           |  L1  |   Y  |   U  |   I  |   O  |   P  |   \    |
+ * |--------+------+------+------+------+------|      |           |      |------+------+------+------+------+--------|
+ * |   ⌫    |   A  |   S  |   D  |   F  |   G  |------|           |------|   H  |   J  |   K  |   L  |; / L2| ' / L⌘ |
+ * |--------+------+------+------+------+------|  L⌘  |           |  L⌘  |------+------+------+------+------+--------|
+ * |  L⇧    |   Z  |   X  |   C  |   V  |   B  |      |           |      |   N  |   M  |   ,  |   .  | / / ⌃|  R⇧    |
```
Comment (ezuk) [inline keyboard/ergodox_ez/keymaps/zweihander-osx/keymap.c]: Epic use of Unicode in this keymap legend!

File: keyboard/ergodox_ez/keymaps/zweihander-osx/keymap.c [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+};
+
+const uint16_t PROGMEM fn_actions[] = {
+    [1] = ACTION_LAYER_TAP_TOGGLE(SYMB)                // FN1 - Momentary Layer 1 (Symbols)
+};
+
+const macro_t *action_get_macro(keyrecord_t *record, uint8_t id, uint8_t opt)
+{
+  // MACRODOWN only works in this function
+      switch(id) {
```
Comment (ezuk) [inline keyboard/ergodox_ez/keymaps/zweihander-osx/keymap.c]: If you're not using a macro, you can remove this whole thing. Just saying :)

File: keyboard/ergodox_ez/keymaps/zweihander-osx/readme.markdown [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+This layout is tuned for people who…
+
+
+## use OS X
+
+This layout doesn’t make any concessions for Windows use. While it should work fine in Windows, I’m not about to devote precious keyboard space to Windows-specific shortcuts.
+
+
+## mouse left-handed while doing other things with the right hand
```
Comment (ezuk) [inline keyboard/ergodox_ez/keymaps/zweihander-osx/readme.markdown]: +1 to this, I'm a lefty too :)

**Comment by ezuk** [comment]
Amazing! :) Love it. Merging this now, feel free to open a new PR if you want to remove that unused macro function just to clean things up there.

File: keyboard/ergodox_ez/keymaps/zweihander-osx/keymap.c (hunk 2 above)
Comment (adiabatic) [inline keyboard/ergodox_ez/keymaps/zweihander-osx/keymap.c]: I don't want it now, but I might in the future and I'm in no rush to delete things. Should I expect any performance or stability improvements if I delete it or comment it out?

File: keyboard/ergodox_ez/keymaps/zweihander-osx/keymap.c (hunk 1 above)
Comment (adiabatic) [inline keyboard/ergodox_ez/keymaps/zweihander-osx/keymap.c]: Thanks; the Unicode equivalents are nice and compact, and space is at a premium…

File: keyboard/ergodox_ez/keymaps/zweihander-osx/keymap.c (hunk 2 above)
Comment (ezuk) [inline keyboard/ergodox_ez/keymaps/zweihander-osx/keymap.c]: nope; just a matter of making the keymap more readable to others who may wonder where it's used. not crucial :)

**Pull request merged**